src/specializations/Plane.jl

```diff
@@ -91,7 +91,9 @@ function integral(
     # Create a wrapper that returns only the value component in those units
     uintegrand(uv) = Unitful.ustrip.(integrandunits, integrand(uv))
     # Integrate only the unitless values
-    value = HCubature.hcubature(uintegrand, -ones(FP, 2), ones(FP, 2); rule.kwargs...)[1]
+    a = 0 .- _ones(FP, 2)
```

Comment:
```suggestion
    a = .- _ones(FP, 2)
```
I think you don't need the zero and it looks a bit odd to me.

Reply:
Didn’t realize that worked.